admin-core/assets/src/dashboard-app/pages/blocks/GlobalBlockStyleExtension.js

```diff
@@ -10,60 +10,58 @@ function classNames( ...classes ) {
     return classes.filter( Boolean ).join( ' ' )
 }
 
-const DynamicContentExtension = () => {
-
-    const enableDynamicContentExtension = useSelector( ( state ) => state.enableDynamicContentExtension );
+const GBSExtension = () => {
+    const enableGBSExtension = useSelector( ( state ) => state.enableGBSExtension );
     const dispatch = useDispatch();
 
-    const dynamicContentStatus = true;
-    // const dynamicContentStatus = 'disabled' === enableDynamicContentExtension ? false : true;
+    const GBSStatus = 'disabled' === enableGBSExtension ? false : true;
 
     useEffect( () => {
 
-        // Create an object with the security and enableDynamicContentExtension properties
+        // Create an object with the security and enableGBSExtension properties
         const data = {
-            security: uag_react.enable_dynamic_content_nonce,
-            value: enableDynamicContentExtension,
+            security: uag_react.enable_gbs_extension_nonce,
+            value: enableGBSExtension,
         };
 
         // Call the getApiData function with the specified parameters
         const getApiFetchData = getApiData( {
             url: uag_react.ajax_url,
-            action: 'uag_enable_dynamic_content',
+            action: 'uag_enable_gbs_extension',
             data,
         } );
 
         // Wait for the API call to complete, but perform no actions after it finishes
         getApiFetchData.then( () => {} );
 
-    }, [enableDynamicContentExtension] );
+    }, [enableGBSExtension] );
 
-    const updatedynamicContentStatus = () => {
+    const updateGBSStatus = () => {
 
         let assetStatus;
-		if ( enableDynamicContentExtension === 'disabled' ) {
+		if ( enableGBSExtension === 'disabled' ) {
             assetStatus = 'enabled';
 		} else {
             assetStatus = 'disabled';
 		}
 
-        dispatch( {type: 'UPDATE_ENABLE_DYNAMIC_CONTENT_EXTENSION', payload: assetStatus } );
+        dispatch( {type: 'UPDATE_GBS_EXTENSION', payload: assetStatus } );
```

Comment:
Minor Coding Practice Improvement.
```suggestion
        dispatch( { type: 'UPDATE_GBS_EXTENSION', payload: assetStatus } );
```

Reply:
Done